packages/dds/tree/src/test/edit-manager/editManager.spec.ts

```diff
@@ -200,373 +201,265 @@ function asDelta(intentions: number[]): Delta.Root {
-        };
-        const c3: TestCommit = {
-            sessionId: peerSessionId1,
-            seqNumber: brand(3),
-            refNumber: brand(0),
-            changeset: TestChangeRebaser.mintChangeset([1, 2], 3),
-        };
-        assert.deepEqual(manager.addSequencedChange(c1), asDelta([1]));
-        assert.deepEqual(manager.addSequencedChange(c2), asDelta([2]));
-        assert.deepEqual(manager.addSequencedChange(c3), asDelta([3]));
-        assert.deepEqual(anchors.intentions, [1, 2, 3]);
-        checkChangeList(manager, [1, 2, 3]);
-    });
+/** Represents the sequencing of a local change. */
+interface UnitTestAckStep {
+    type: "Ack";
+    /**
+     * The sequence number for this change.
+     * Should match the sequence number of the oldest `UnitTestPushStep`
+     * for which there is no `UnitTestAckStep` step.
+     */
+    seq: number;
+}
 
-    it("Can rebase a single peer change over multiple peer changes", () => {
-        const { manager, anchors } = editManagerFactory();
-        const c1: TestCommit = {
-            sessionId: peerSessionId1,
-            seqNumber: brand(1),
-            refNumber: brand(0),
-            changeset: TestChangeRebaser.mintChangeset([], 1),
-        };
-        const c2: TestCommit = {
-            sessionId: peerSessionId1,
-            seqNumber: brand(2),
-            refNumber: brand(1),
-            changeset: TestChangeRebaser.mintChangeset([1], 2),
-        };
-        const c3: TestCommit = {
-            sessionId: peerSessionId1,
-            seqNumber: brand(3),
-            refNumber: brand(2),
-            changeset: TestChangeRebaser.mintChangeset([1, 2], 3),
-        };
-        const c4: TestCommit = {
-            sessionId: peerSessionId2,
-            seqNumber: brand(4),
-            refNumber: brand(0),
-            changeset: TestChangeRebaser.mintChangeset([], 4),
-        };
-        assert.deepEqual(manager.addSequencedChange(c1), asDelta([1]));
-        assert.deepEqual(manager.addSequencedChange(c2), asDelta([2]));
-        assert.deepEqual(manager.addSequencedChange(c3), asDelta([3]));
-        assert.deepEqual(manager.addSequencedChange(c4), asDelta([4]));
-        assert.deepEqual(anchors.intentions, [1, 2, 3, 4]);
-        checkChangeList(manager, [1, 2, 3, 4]);
-    });
+/** Represents the sequencing of a peer change */
+interface UnitTestPullStep {
+    type: "Pull";
+    /** The sequence number for this change. */
+    seq: number;
+    /**
+     * The sequence number of the latest change that the issuer of this change knew about
+     * at the time they issued this change.
+     */
+    ref: number;
+    /** The ID of the peer that issued the change. */
+    from: SessionId;
+    /**
+     * Ordered list of sequence numbers that correspond to the net change for the local document.
+     * This is required to make tests easier to read and debug. The `unitTest` function verifies
+     * that those expectations are correct.
```

Comment:
```suggestion
     * The delta which should be produced by the EditManager when it receives this change.
     * This information is derived by the `unitTest` function, but is explicitly provided
     * to make tests easier to read and debug.
```

I would also consider making this field optional.